packages/next/src/build/output/index.ts

```diff
@@ -138,10 +139,7 @@ buildStore.subscribe((state) => {
     appUrl: appUrl!,
     loading: false,
     typeChecking: false,
-    partial:
-      clientWasLoading && (serverWasLoading || edgeServerWasLoading)
-        ? 'client and server'
-        : undefined,
+    partial: undefined,
```

Comment:
It seems this property is unused as it's never set to anything other than `undefined` 🤔